### DISCUSSION TITLE
does transforms are applied to all channels

### DISCUSSION THREAD

**Discussion by jakubMitura14** [Project-MONAI/MONAI#4661] [Q&A] [answered]
hello I have 3 channel 3d image (so 4d tensor where first dimension is channel) does transforms like for example rician noise etc will be applied to all channels? 

Thanks!

**Comment by Nic-Ma** [comment, marked as answer]
Hi @jakubMitura14 ,

Yes, the transforms will apply to all the channels, some intensity transforms also have the `channel-wise` arg to apply transform for every channel separately.

Thanks.
